jmix-flowui/flowui/src/main/java/io/jmix/flowui/xml/layout/loader/component/AbstractGridLoader.java

```diff
@@ -255,7 +244,10 @@ protected void loadEditorActionsColumn(T resultComponent, Element columnElement)
 
         editColumn.setEditorComponent(actions);
 
-        loadString(columnElement, "key", editColumn::setKey);
+        //If the key is null then NPE will rise when the settings are applied
+        loadString(columnElement, "key").ifPresentOrElse(
+                editColumn::setKey,
+                () -> editColumn.setKey(EDITOR_ACTION_COLUMN_DEFAULT_KEY));
```

Comment:
Let's use element name as Id. Having element name in plural form and id in singular looks weird